website_sale_suggest_create_account/static/tests/tours/checkout.esm.js

```diff
@@ -0,0 +1,45 @@
+import {registry} from "@web/core/registry";
+registry.category("web_tour.tours").add("shop_buy_checkout_suggest_account_website", {
+    test: true,
+    url: "/shop",
+    steps: () => [
+        // Shop Page
+        {
+            trigger: "a:contains('Customizable')",
```

Comment:
```suggestion
            trigger: "a:contains('Customizable')",
            run: "click",
```

This is not failing because of the above. In many steps the action is missing.